Sources/Nimble/Matchers/BeginWithPrefix.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+/// A Nimble matcher that succeeds when the exepected sequence is a prefix of the actual sequence.
+///
+/// This is a matcher abstraction for https://developer.apple.com/documentation/swift/sequence/2854218-starts
+public func beginWith<Seq1: Sequence, Seq2: Sequence>(prefix expectedPrefix: Seq2?)
+    -> Predicate<Seq1> where Seq1.Element: Equatable, Seq1.Element == Seq2.Element {
+    return Predicate.define("beginWith <\(stringify(expectedPrefix))>") { (actualExpression, msg) in
```

Comment:
```suggestion
    return Predicate.define("begin with <\(stringify(expectedPrefix))>") { (actualExpression, msg) in
```

to match with the existing ones: https://github.com/Quick/Nimble/blob/ca652f9a6e4c4190e40d83469c2b8df9c2c4bfaf/Sources/Nimble/Matchers/BeginWith.swift#L5-L6